tests/unit/test_habit.py

```diff
@@ -7,27 +7,56 @@
 def test_create_habit(client: TestClient, user_1: User):
     response = client.post(
         "/habits/",
-        json={"text": "Read quality Python Code", "user_id": user_1.id},
+        json={"text": "Read quality Python code", "user_id": user_1.id},
     )
     data = response.json()
 
     assert response.status_code == 201
-    assert data["text"] == "Read quality Python Code"
+    assert data["text"] == "Read quality Python code"
     assert data["user_id"] == 1
     assert data["id"] is not None
 
 
 def test_create_habit_bad_user(client: TestClient):
     response = client.post(
         "/habits/",
-        json={"text": "Read quality Python Code", "user_id": 0},
+        json={"text": "Read quality Python code", "user_id": 0},
     )
     data = response.json()
 
     assert response.status_code == 400
     assert data["detail"] == "Not a valid user id"
 
 
+def test_create_habit_duplicate_same_user(client: TestClient, user_1_with_habits: User):
+    response = client.post(
```

Comment:
yep this makes more sense, I might update mine to take the user into account as well, not sure yet